server/controllers/auth/strategies/facebookStrategy.js

```diff
@@ -68,7 +68,7 @@ class Facebook {
        }
     }).spread((user, created) => {
       const token = JWTHelper.generateToken(
-        removeDateStampAndPassword(user.dataValues)
+        removeDateStampAndPassword(user.dataValues), '1d'
```

Comment:
Nice work on this @gbols. Am confused though. At this point, you are hard-coding the expiration time but at another point, you assigned the expiration time to a variable. It would be cool if you used one specific approach